modules/microshift-lvms-csi-snapshot-optional.adoc

```diff
@@ -0,0 +1,14 @@
+// Module included in the following assemblies:
+//
+// * microshift_storage/microshift-storage-plugin-overview.adoc
+
+:_mod-docs-content-type: CONCEPT
```

Comment:
All of this content should be at the end of the assembly, not before the regular use, as this is an edge case, to my knowlege.

Reply:
checking with @copejon on the position of the the modules in the assembly.

Reply:
Jon has given lgtm on the PR